label_studio/users/templates/users/user_account.html

```diff
@@ -176,9 +176,13 @@
   <script>
     (() => {
       {% if settings.HOSTNAME %}
-      const CURL_TEMPLATE = "curl -X GET {{ settings.HOSTNAME }}/api/projects/ -H 'Authorization: Token'"
+      const hostname = {{ settings.HOSTNAME }};
       {% else %}
-      const CURL_TEMPLATE = "curl -X GET http://localhost:8080/api/projects/ -H 'Authorization: Token'"
+      const hostname = 'http://localhost:8080';
+      {% endif %}
+      const CURL_TEMPLATE = "curl -X GET {{ settings.HOSTNAME }}/api/projects/ -H 'Authorization: Token"
+      {% else %}
+      const CURL_TEMPLATE = "curl -X GET http://localhost:8080/api/projects/ -H 'Authorization: Token"
```

Comment:
`CURL_TEMPLATE` isn't necessary anymore. These lines can be removed.

Reply:
why is the curl template not necessary?

Reply:
`CURL_TEMPLATE` was being used to set `example.value` on line 205 before this change. Now example.value is being constructed using the `hostname` const.

Reply:
updated @wesleylima